pwa/src/components/componentCard/ComponentCard.module.css

```diff
@@ -67,7 +67,7 @@
   --web-app-tag-background-color: #dd3c49;
   --web-app-tag-color: var(--web-app-color-white);
 }
-.procesCategory .processCategory {
+.procesCategory, .processCategory {
```

Comment:
For readability, I'd opt to put comma separated rules underneath each other, like so:

```
.process,
.other {
  ...
}
```